src/main/java/racingcar/ui/InputView.java

```diff
@@ -0,0 +1,62 @@
+package racingcar.ui;
+
+import java.util.Scanner;
+
+public class InputView {
+    private final static Scanner scanner = new Scanner(System.in);
```

Comment:
접근제어자 순서도 [컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)에 따라 변경해 주시면 좋을 것 같아요
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```